gdk4-wayland/Cargo.toml

```
@@ -14,6 +14,7 @@ rust-version = "1.64"
 
 [features]
 v4_4 = ["ffi/v4_4"]
+v4_10 = ["ffi/v4_10"]
```

```suggestion
v4_10 = ["ffi/v4_10", "v4_4"]
```
also the new feature should be added to the README file

---

This wasn't fixed?